sklearn/metrics/pairwise.py

```diff
@@ -932,17 +968,20 @@ def paired_distances(X, Y, *, metric="euclidean", **kwds):
 
     Parameters
     ----------
-    X : ndarray (n_samples, n_features)
+    X : ndarray (n_samples, n_features) or, (n_samples, 2)
```

Comment:
I don't think this needs changing. For haversine, n_features=2, but that doesn't mean the general case is insufficient